### ISSUE TITLE
Syncing alt text changes in block editor back to media library

### ISSUE THREAD

**Issue by gaambo** [goodwp/altinator-wp#3]
Related to #2 

## Use Case 1: Inserting an existing image with an existing alt text
When inserting an image **with** an existing alt-text from the media-library into a block, the existing alt-text will be shown in the blocks settings in the sidebar. When this "local" alt-text is changed, it is not synced back to the media library. 
Which can be what the user wants in some cases, but in some cases not. We should give the user the choice and add a button to sync the alt-text back to the media library. A short explanation that otherwise only the "local" alt text is changed, should be added as well.

## Use Case 2: Inserting an existing image without an alt text
When inserting an image **without** an existing alt-text from the media-library into a block, and the user then adds a "local" alt-text to the image block, it is not synced back to the media library. 
Which can be what the user wants in some cases, but in some cases not. We should give the user the choice and add a button to sync the alt-text back to the media library. A short explanation that otherwise only the "local" alt text is changed, should be added as well.

## Use Case 3: Uploading a new image into the block editor and adding alt text to the block
When uploading a new image directly into the block editor without setting an alt text in the media-library popup, but later adding a "local" alt-text to the image block, it is not synced back to the media library. 
Which can be what the user wants in some cases, but in some cases not. We should give the user the choice and add a button to sync the alt-text back to the media library. A short explanation that otherwise only the "local" alt text is changed, should be added as well.

Reported by @00travelgirl00

**Notes:**
"Image block" means all blocks that we want to support. That will probably be `core/image`, `core/media-text`, `core/cover`(?),... but could/should be expandable.
